plaidml/bridge/openvino/ops/detection_output.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace plaidml::edsl;
+
+namespace PlaidMLPlugin {
+
+edsl::Tensor DecodeBoxes(edsl::Tensor priors, edsl::Tensor variances, edsl::Tensor location, std::string code_type,
+                         int input_height, int input_width, int batch, int num_priors, bool clip_before_nms) {
+  edsl::Tensor decoded_bboxes;
+  if (code_type == "caffe.PriorBoxParameter.CORNER") {
+    decoded_bboxes = priors / input_width + variances * location;
+  } else if (code_type == "caffe.PriorBoxParameter.CENTER_SIZE") {
+    auto var_xmin = op::slice(variances).add_dim(0, batch).add_dim(0, num_priors).add_dim(0, 1);
```

Comment:
For these slices, can we use mask to avoid these ops?  like https://github.com/plaidml/plaidml/pull/1736/commits/f7cfa91c87667b525ac4c55093dd6c86818d0b83.   The reason i revert this commit is i have to use maximum and minimum so it will be a little slow, Not sure if we can change here.

Reply:
Seems using mask does not make this faster. Mark it as a note in case any there are changes in the future.